dashboard/src/app/ide/ide.service.js

```diff
@@ -96,38 +99,48 @@ class IdeSvc {
     this.$log.error(error);
   }
 
-  startIde(noIdeLoader) {
+  startIde(workspace, noIdeLoader) {
```

Comment:
I don't get why there is a new workspace parameter as there is already a class attribute selectedWorkspace